wrappingGsdk/wrapper/Program.cs

```diff
@@ -38,8 +44,42 @@ static void Main(string[] args)
         // starts main game process and wait for it to complete
         public static void InitiateAndWaitForGameProcess(string gameserverExe, IEnumerable<string> args)
         {
-             // here we're starting the script that initiates the game process
-            gameProcess = StartProcess(gameserverExe, args);
+            activeConfig = GameserverSDK.getConfigSettings();
+
+            // here we're starting the script that initiates the game process
+            if (activeConfig.TryGetValue(portName, out string listeningPortString))
```

Comment:
can you please add a comment here for the difference in port between process and container?

Reply:
Thank you so much for the review, I applied feedbacks and updated it.